src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.test.tsx

```diff
@@ -109,26 +120,33 @@ describe('<CheckboxGrid />', () => {
       }
     });
 
-    expect(wrapper.find(Checkbox).length).toBe(totalCheckedCheckboxes);
+    expect(container.querySelectorAll('.defaultCheckbox').length).toBe(
+      totalCheckedCheckboxes
+    );
   });
 
   it('hides the title when hideTitles is true', () => {
-    wrapper = mount(<CheckboxGrid {...defaultProps} hideLabel={true} />);
+    const { container } = renderCheckboxGrid({ hideLabel: true });
 
-    expect(wrapper.find('.checkboxGridTitle').length).toBe(0);
+    expect(container.querySelectorAll('.checkboxGridTitle').length).toBe(0);
```

Comment:
would `container.querySelector('.checkboxGridTitle')).toBeFalsy()` communicate the intention better?

Reply:
That's done